src/rgw/rgw_rest_swift.cc

```diff
@@ -92,6 +92,9 @@ static void dump_account_metadata(struct req_state * const s,
   dump_header(s, "X-Account-Object-Count", global_stats.objects_count);
   dump_header(s, "X-Account-Bytes-Used", global_stats.bytes_used);
   dump_header(s, "X-Account-Bytes-Used-Actual", global_stats.bytes_used_rounded);
+  if (s->cct->_conf->rgw_swift_accept_ranges) {
```

Comment:
I'm not a big fan of introducing a brand new configurable for that. Instead, I would reuse the already existing `rgw_swift_enforce_content_length` and dump just the `Accept-Ranges` header when it's `false`.

Also, `dump_account_metadata` and `dump_container_metadata` aren't the best places for that IMO. The most maintainable way would be to extend the current checks for `rgw_swift_enforce_content_length`, I think.

Reply:
Thanks a lot for the comments. :)

I will take into account the comments and update the patch ASAP.